src/Integrations/Integrations/IntegrationsLoader.php

```diff
@@ -10,7 +10,7 @@
 use DDTrace\Integrations\Guzzle\GuzzleIntegration;
 use DDTrace\Integrations\Laravel\LaravelIntegration;
 use DDTrace\Integrations\Lumen\LumenIntegration;
-use DDTrace\Integrations\Memcached\MemcachedIntegration;
```

Comment:
Why do we replace the old one? Shouldn't we just add the new one?

Reply:
I see, the refactoring functionality of PHPStorm replaced too many things. Well spotted. (And no, the new one should not be added, that's for PHP 5.)